zilliqa/src/api/types/eth.rs

```diff
@@ -48,6 +48,24 @@ impl QuorumCertificate {
     }
 }
 
+#[derive(Clone)]
+pub enum ErrorCode {
```

Comment:
The codes are different, I believe - happy to simply merge them, but you'll get quite a lot of "ZilErr ... " and "EthErr". I'd resist referencing symbolic types, because the error codes are defined by the API in numeric terms and if the symbolic constants ever change, that will break our APIs by reference.

Let me know what you prefer - I don't feel strongly ..

Reply:
Okay fair enough, happy to stick with this for now if you are :)